docker/Dockerfile.ldk

```
@@ -0,0 +1,49 @@
+FROM ubuntu:22.04
+LABEL maintainer="Prakhar Saxena prakharrsaxena@gmail.com"
+
+ENV TZ=Europe/Minsk
+ENV BITCOIN_VERSION=23.0
+
+# Set timezone
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Install Ubuntu utilities and essential dependencies
+RUN apt-get update && apt-get install -y \
+    software-properties-common \
+    build-essential \
+    curl wget jq \
+    git python3 python3-pip
+
+# Install Rust
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+RUN echo 'source $HOME/.cargo/env' >> $HOME/.bashrc
+
+# Install Bitcoin Core
+RUN cd /tmp/ && \
+    wget https://bitcoincore.org/bin/bitcoin-core-$BITCOIN_VERSION/bitcoin-$BITCOIN_VERSION-x86_64-linux-gnu.tar.gz -O bitcoin.tar.gz && \
+    tar -xvzf bitcoin.tar.gz && \
+    mv /tmp/bitcoin-$BITCOIN_VERSION/bin/bitcoin* /usr/local/bin/ && \
+    rm -rf bitcoin.tar.gz /tmp/bitcoin-$BITCOIN_VERSION
+
+# Clone and build LDK-Sample
+RUN git clone https://github.com/Psycho-Pirate/ldk-sample.git
+RUN cd ldk-sample && cargo build
+
+# Install runner
+RUN pip install -U poetry
+RUN pip install ldk-lnprototest
+
+# Set PYTHONPATH environment variable
+ENV LDK_SRC="/ldk-sample"
+
+# Set workdir and copy project files
+RUN mkdir lnprototest
+COPY . lnprototest
+
+RUN cd lnprototest && \
+    poetry config virtualenvs.create false && \
+    poetry install
+RUN cd lnprototest && ls -lha
+
+CMD ["./lnprototest/docker/entrypoint.sh"]
```

```suggestion
CMD ["./lnprototest/docker/entrypoint.sh"]
```

why not keep the script simple and add a `ldk-entrypoint.sh`? IIRC we do not have any constrain about the name of the script that we will use.